package.json

```diff
@@ -1,64 +1,17 @@
 {
     "name": "kaetram",
     "version": "1.2.1",
-    "private": false,
+    "private": true,
```

Comment:
Why is it private?

Reply:
This is private because this doesn't actually get published to npm.  The client,server,common packages get published instead.  The package.json really is just to have lerna and manage the dependency chain

Reply:
yes, makes sense